tests/js/spec/views/issueList/tagFilter.spec.jsx

```diff
@@ -11,9 +11,9 @@ describe('IssueListTagFilter', function() {
     MockApiClient.clearMockResponses();
     project = TestStubs.ProjectDetails();
 
-    tagValueLoader = () => {
```

Comment:
This seems like something lintable, can we make this a TSC proposal instead of one off changes?

Reply:
https://github.com/getsentry/eslint-config-sentry/pull/58